packages/ui-components/src/__tests__/TransactionManager.test.ts

```
@@ -0,0 +1,211 @@
+			});
+
+			await manager.createRemoveOrderTransaction({
+				subgraphUrl: 'https://api.example.com',
+				txHash:
+					'0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef' as `0x${string}`,
+				orderHash:
+					'0xabcdef1234567890abcdef1234567890abcdef1234567890abcdef1234567890' as `0x${string}`,
+				chainId: 1
+			});
+
+			onError!();
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)